cmd/cli/cmd/alpha/content/helpers.go

```diff
@@ -0,0 +1,29 @@
+package content
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+func writeManifestFiles(files map[string]string) error {
```

Comment:
I think it would be better to fail if files in a given path already exist. Someone can override already generated and adjusted manifests with new command run.

BTW And please move it out of this directory into `internal`. Let's have just files which are related to commands only.

Reply:
I added a `--override` flag, which is set to `false` by default.